src/services/create-project.service.js

```diff
@@ -61,9 +64,12 @@ export default (
 
   const id = slug(projectName).toLowerCase();
 
-  const path = `${parentPath}/${id}`;
+  // For Windows Support
+  // To support cross platform with slashes and escapes
+  // Taken from https://github.com/AWolf81/guppy/commit/b2434d907d0c2c2585006d82ef14523a974de6a0
```

Comment:
Since this is now your PR, I think you can omit this line? I think it was nice to add it when Benny had made the change, to credit your work, but now this PR is your work.

Reply:
You're right - I'll remove the last line of the comment.